### ISSUE TITLE
Airdrop closed?

### ISSUE THREAD

**Issue by keparatkamubangsat** [keybase/keybase-issues#3662]
Hello admin airdrop closed? I can't join airdrop. Notice : You're not qualified for airdrop!

**Comment by Patol11** [comment]
Many users have this problem. There is probably no admin here. It looks like a big scam to me.

**Comment by Zero-1729** [comment]
It's not a scam, it seems a significant number of users were just excluded from the airdrop, most likely because they mimicked spam-like behaviour: attempting to register for the airdrop multiple times, creating multiple accounts on a single device, etc.

**Comment by loseirdo** [comment]
> most likely because they mimicked spam-like behaviour

While this _may_ be the case for some accounts, it is _not_ the case for the majority of them. It seems the issue is almost entirely centered around what cell phone carrier you have. If you register with a phone number on a virtual network (not one of the four major networks), it seems the app recognizes that you have a number in one of the approved countries, but rejects your attempt to join anyway. Other users have had success joining the spacedrop after removing a VOIP number associated with their account.

Unfortunately, with absolutely _no_ feedback from the dev team, this is still largely speculation, and it seems that the only workaround is to pay for a new phone line with one of the major carriers, which is certainly not worth it (especially with no guarantee that it will even work).

I agree that it's not a scam, but it's frustrating to see just how bungled this entire thing has been, with zero word from any of the developers about it.

**Comment by Zero-1729** [comment]
>  it is not the case for the majority of them ...

I can testify to that, as I too face this problem.

> Unfortunately, with absolutely no feedback from the dev team, this is still largely speculation ...

Agreed. It sucks that there has been no feedback or at least clarification from Keybase. This issue is being experienced by a large number of users, yet, they have failed to provide any form of clarification. I hope that they do provide some explanation or clarification.

I mean, they should have anticipated that there would be attempts by people to scam their way to get the drop, as It is commonplace in airdrops. The previous requirement of having a GitHub account created before September was really good IMO. Stellar should have known that the addition of phone numbers as a requirement would attract a lot more scammers. Unfortunately, now we are all suffering the consequence of that decision.

**Comment by Patol11** [comment]
OK, this is not a scam, but the lack of any answers from Keybase in  is puzzling. My verified number is not virtual, and I did not qualify for the airdrop.

**Comment by Zero-1729** [comment]
True, it is really unsettling that there is still no word on this issue yet. So really it could be that you were blacklisted for some weird reason only Keybase or Stellar know.

**Comment by Stellaruser** [comment]
Ich denke schon das es Betrug ist!! Airdrop ist ein Fake! 
 Wenn man sich mal anschaut was dieses Keybase übehaupt ist, da sind ja Google, Facebook & Co ja Waisenknaben dagegen!
Sie wollen nur Daten und alle Verknüpfungen was ihr alles macht. Das ist die ultimative Datensammlung! Der vollkommene gläserne Bürger! Die wissen über Euch besser Bescheid als Ihr selbst!
